src/utils/ProfileUtils.ts

```diff
@@ -56,6 +58,22 @@ async function extractConfigsToImportedProfile(
     }
 }
 
+export async function installModsAndResolveConflicts(
+    downloadedMods: ThunderstoreCombo[],
+    profile: ImmutableProfile,
+    store: Store<any>
+): Promise<void> {
+    await ProfileModList.requestLock(async () => {
+        try {
+            const modList: ManifestV2[] = await installModsToProfile(downloadedMods, profile);
+            await store.dispatch('profile/updateModList', modList);
+            throwForR2Error(await ConflictManagementProvider.instance.resolveConflicts(modList, profile));
+        } catch (e: any) {
```

Comment:
Minor: IDK why I didn't notice this earlier, probably due to the long comment above. Anyways, We now catch the exception to throw it as-is again, so basically the try-catch does nothing here. I'll leave it up to you if/when it should be fixed.